pages/settings/profile.tsx

```diff
@@ -147,14 +160,46 @@ export default function Settings(props) {
                     First Day of Week
                   </label>
                   <div className="mt-1">
-                    <select
+                    <Select
                       id="weekStart"
                       value={selectedWeekStartDay}
-                      onChange={(e) => setSelectedWeekStartDay(e.target.value)}
-                      className="shadow-sm focus:ring-blue-500 focus:border-blue-500 mt-1 block w-full sm:text-sm border-gray-300 rounded-md">
-                      <option value="Sunday">Sunday</option>
-                      <option value="Monday">Monday</option>
-                    </select>
+                      onChange={setSelectedWeekStartDay}
+                      className="shadow-sm focus:ring-blue-500 focus:border-blue-500 mt-1 block w-full sm:text-sm border-gray-300 rounded-md"
+                      options={[
+                        { value:'Sunday', label:'Sunday' },
+                        { value:'Monday', label:'Monday' }
+                      ]} />
+                  </div>
+                </div>
+                <div>
+                  <label htmlFor="theme" className="block text-sm font-medium text-gray-700">
+                    Single Theme
+                  </label>
+                  <div className="my-1">
+                    <Select
+                      id="theme"
+                      isDisabled={!selectedTheme}
+                      defaultValue={selectedTheme || themeOptions[0]}
+                      onChange={setSelectedTheme}
+                      className="shadow-sm focus:ring-blue-500 focus:border-blue-500 mt-1 block w-full sm:text-sm border-gray-300 rounded-md"
+                      options={themeOptions} />
```

Comment:
Codacy has a fix for the issue: [Insert `⏎···················`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
                      options={themeOptions}
```